assets/scripts/progress.js

```diff
@@ -27,5 +31,23 @@ function numberWithCommas(x) {
     (targetUSD / parseInt(ethUsdConversion)).toFixed(3)
   );
 
-  document.getElementById("percent").textContent = `${percentage.toFixed(0)}%`
+  document.getElementById("percent").textContent = `${percentage.toFixed(0)}%`;
+
+  for (var i = 0; i < milestoneMillions.length; i++) {
+    console.log(targetUSD);
+    console.log(milestoneMillions[i]);
+    const percentage = milestoneMillions[i] / targetUSD;
+
+    const marker = document.createElement("div");
+    marker.className = "milestone";
+    marker.style.left = percentage * 100 + "%";
+
+    const label = document.createElement("div");
+    label.className = "milestone-label";
+    label.style.left = percentage * 100 + "%";
+    label.innerText = "Goal " + (i + 1);
```

Comment:
Can we make these say $14mm and $30mm perhaps?

Reply:
It'd also be helpful to provide brief summaries of the milestones underneath the bar. Something like:

Goal 1 ($14M): Participate in the auction
Goal 2 ($20M): Have more than the estimated cost of the constitution
Goal 3 ($30M): Have enough to be competitive at the auction

And strikethrough all three lines to show we've beat them.

Another idea I had was to color code the goal lines on the bar as well as the summaries to make things even clearer.

Reply:
One more idea - have the targetUsd always be a certain percentage or amount higher than raisedUsd so that the progress bar automatically updates with a consistent and predictable look.